Modules/SubModule_ASL/xASL_wrp_Quantify.m

```diff
@@ -73,6 +73,18 @@ function xASL_wrp_Quantify(x, PWI_Path, OutputPath, M0Path, SliceGradientPath)
 %% 1.   Load PWI
 fprintf('%s\n','Loading PWI & M0 images');
 
+if isfield(x,'UseBasilQuantification') & x.UseBasilQuantification
```

Comment:
Happy to change - my understanding is that the only difference is short-circuiting of the AND so it's really a style issue. I thought I'd seen & used generally in the codebase, but looking again in most cases && seems to be the norm so will switch to this.

Reply:
`&` is more flexible, `&&` is specific for booleans and will also first check the first argument, and if it is not true it will skip checking the second argument. `&` I believe will first try to "multiply" the two arguments, which would crash if this field doesn't exist.